website/src/components/Changelog/GUI.tsx

```diff
@@ -28,6 +22,9 @@ export default function GUI({ title }: { title: string }) {
               reported as "IPC connection closed"
             </Link>{" "}
           </li>
+          {title === "Linux GUI" && (
+          )}
```

Comment:
This is the same as just removing it right?

```suggestion
```

Reply:
Yeah but I figured I'd leave it in case anything gets fixed on Linux before the next Client release